src/toolbarContext/toolbarContext.ts

```diff
@@ -0,0 +1,27 @@
+import { Consumer, Context, createContext, Provider } from 'react';
+
+import { TokenSorting } from '@app/api/graphQL/tokens/types';
+import { SelectOptionProps, ViewType } from '@app/components';
+
+export type ToolbarContextProps = {
```

Comment:
let's get rid of the context: sort, filter and search values are written in the navigation bar. let's add a view there as well and not duplicate data